packages/react-ui-core/src/AutoSuggestField/InputAnchor.js

```diff
@@ -0,0 +1,64 @@
+import React, { PureComponent } from 'react'
```

Comment:
This file does exactly what the [AnchorButton](https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Dropdown/AnchorButton.js) does.  Can we please try to use that component and modify it so it will work for both an input and button?

Reply:
Fair enough. So give `AnchorButton` a prop to be able to be a `Field` optionally?